CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- GCS endpoint to download UDP gateway configuration file.
```

Comment:
You need to explicitly mention the new config option that operators need to set on their deployments when upgrading, otherwise their deployments will point to localhost.